spring-cloud-starter-dataflow-server-local/pom.xml

```diff
@@ -33,6 +33,11 @@
 			<version>1.5.5</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
In `LocalServerSecurityWithUsersFileTests` I use the Guava test dependency for creating Maps, e.g. to define test request parameter:

```
ImmutableMap.of("start", "2", "detailLevel", "2")
```